public/components/datasources/components/manage/associated_objects/modules/associated_objects_table.tsx

```diff
@@ -134,7 +142,22 @@ export const AssociatedObjectsTable = (props: AssociatedObjectsTableProps) => {
           ),
           type: 'icon',
           icon: 'discoverApp',
-          onClick: (item: AssociatedObject) => console.log('Discover', item),
+          onClick: (asscObj: AssociatedObject) => {
+            if (asscObj.type === 'covering' || asscObj.type === 'materialized') {
+              // find the flint index name through the cached acceleration
+              const acceleration = cachedAccelerations.find(
+                (acc) => getAccelerationName(acc.indexName, acc, datasourceName) === asscObj.name
+              );
+              redirectToExplorerOSIdx(acceleration!.flintIndexName);
+            } else if (asscObj.type === 'table') {
+              redirectToExplorerWithDataSrc(
+                asscObj.datasource,
+                's3glue',
```

Comment:
Is this a constant declared somewhere in explorer or core for the datasource selector?

Reply:
ah it is, just found where. updating all instances of this string with that constant